stellar-dotnet-sdk-test/RevokeSponsorshipTest.cs

```diff
@@ -0,0 +1,441 @@
+        KeyPair.FromSecretSeed("SDBNUIC2JMIYKGLJUFI743AQDWPBOWKG42GADHEY3FQDTQLJADYPQZTP");
+
+    private const string DataName = "my secret";
+
+    [TestInitialize]
+    public async Task Setup()
+    {
+        Network.UseTestNetwork();
+        // Generates a new _accountId in case the testnet has been reset
+        try
+        {
+            await _server.Accounts.Account(_sponsoringId);
+        }
+        catch (AccountNotFoundException)
+        {
+            _sponsoringAccount = await CreateNewRandomAccountOnTestnet();
+        }
+
+        try
+        {
+            await _server.Accounts.Account(_sponsoredId);
+        }
+        catch (AccountNotFoundException)
+        {
+            _sponsoredAccount = await CreateNewRandomAccountOnTestnet();
+        }
+
+        _sponsoringId = _sponsoringAccount.AccountId;
+        _sponsoredId = _sponsoredAccount.AccountId;
+        _assetA = new AssetTypeCreditAlphaNum4("AAA", _sponsoringId);
+    }
+
+    [TestCleanup]
+    public void Cleanup()
+    {
+        Network.Use(null);
+        _server.Dispose();
+    }
+
+    private async Task<KeyPair> CreateNewRandomAccountOnTestnet()
+    {
+        bool isSuccess;
+        KeyPair account;
+        do
+        {
+            account = KeyPair.Random();
+            var fundResponse = await _server.TestNetFriendBot.FundAccount(account.AccountId).Execute();
+            var result = TransactionResult.FromXdrBase64(fundResponse.ResultXdr);
+            isSuccess = result.IsSuccess && result is TransactionResultSuccess;
+        } while (!isSuccess);
+
+        return account;
+    }
+
+    private async Task RevokeClaimableBalanceSponsorship(string balanceId)
+    {
+        Assert.IsNotNull(balanceId);
+        var account = await _server.Accounts.Account(_sponsoringId);
+
+        var revokeOperation = new RevokeLedgerEntrySponsorshipOperation.Builder(balanceId).Build();
+
+        var beginSponsoringOperation = new BeginSponsoringFutureReservesOperation.Builder(_sponsoringId)
+            .SetSourceAccount(_sponsoredAccount)
+            .Build();
+        var endSponsoringOperation = new EndSponsoringFutureReservesOperation.Builder().Build();
+        
+        var tx = new TransactionBuilder(account)
+            .AddOperation(beginSponsoringOperation)
+            .AddOperation(revokeOperation)
+            .AddOperation(endSponsoringOperation)
+            .Build();
+        tx.Sign(_sponsoringAccount);
+        tx.Sign(_sponsoredAccount);
+
+        var txResponse = await _server.SubmitTransaction(tx);
+        Assert.IsNotNull(txResponse);
+        Assert.IsTrue(txResponse.IsSuccess());
+        Assert.IsNotNull(txResponse.ResultXdr);
+        var transactionResult = TransactionResult.FromXdrBase64(txResponse.ResultXdr);
+        Assert.IsTrue(transactionResult.IsSuccess);
+        Assert.IsInstanceOfType(transactionResult, typeof(TransactionResultSuccess));
+        Assert.AreEqual("0.00003", transactionResult.FeeCharged);
```

Comment:
This can be different every time no need to check this. Please search for this
`, transactionResult.FeeCharged);`
and remove all of them.